baselines/asset/samples/generated/v1/asset_service.batch_get_assets_history.js.baseline

```diff
@@ -17,40 +17,40 @@
 
 function main(parent) {
   // [START cloudasset_v1_generated_AssetService_BatchGetAssetsHistory_async]
-  /**
-   * TODO(developer): Uncomment these variables before running the sample.
-   */
-  /**
-   *  Required. The relative name of the root asset. It can only be an
-   *  organization number (such as "organizations/123"), a project ID (such as
-   *  "projects/my-project-id")", or a project number (such as "projects/12345").
-   */
+  //
+  // TODO(developer): Uncomment these variables before running the sample.
+  //
+  //
+  // Required. The relative name of the root asset. It can only be an
+  // organization number (such as "organizations/123"), a project ID (such as
```

Comment:
We don't need this format for cloud-rad anymore as the processing happens in api-documenter. So sorry for the churn. The rest looks good.